typit.el

```diff
@@ -76,10 +76,10 @@
   '((t (:inherit font-lock-constant-face)))
   "Face used to render statistical values after typing.")
 
-(defcustom typit-dict "english.txt"
+(defcustom typit-dict "deutsch.txt"
   "Name of dictionary file to use."
   :tag  "Dictionary to use"
-  :type '(choice (const :tag "English" "english.txt")))
+  :type '(choice (const :tag "Deutsch" "deutsch.txt")))
```

Comment:
Please add a new option instead of overwriting. Add one more `(const ...)` element to the list starting with the `choice` symbol. Also, AFAIK “Deutsch” means “German” in German, maybe it's better to spell it in English (i.e. `(const :tag "German" "deutsch.txt")`)? Although I'm not sure.